cmd/main.go

```diff
@@ -46,6 +46,7 @@ var (
 	port             = flag.Int("port", 12345, "Port on which to run the server.")
 	useALTS          = flag.Bool("use_alts", false, "Whether to use ALTS server authentication")
 	bigqueryOnly     = flag.Bool("bigquery_only", false, "The service only serves sparql query")
+	statVarMode      = flag.Bool("stavar_mode", false, "Use new storage that is built on top of StatVar instead of StatPop")
```

Comment:
statvar_mode

nit: the g3 impl calls it statvarobs_mode or svobs_mode, since that's the distinguishing thing (we still have statvar in old mode)

Reply:
Done